Config.cmake.in

```diff
@@ -1,5 +1,12 @@
 @PACKAGE_INIT@
 
+# The exported target depends on targets from these dependencies,
+# so we need to find them first.
+include(CMakeFindDependencyMacro)
+find_dependency(ZLIB)
+find_dependency(JPEG)
+find_dependency(PNG)
```

Comment:
These are not included in any public header so not directly needed.
For linking I thought CMake would handle these for us?

Reply:
The targets provided by these libraries are used in the Targets file CMake creates, but are not included by it. This means that any external targets we use have to be discovered before including the Targets file, or they will be undefined.